packages/functional-tests/tests/oauth/oauthPromptNone.spec.ts

```diff
@@ -0,0 +1,253 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { test, expect } from '../../lib/fixtures/standard';
+
+let email;
+const password = 'passwordzxcv';
+
+test.describe('oauth prompt none', () => {
+  test.beforeEach(async ({ pages: { login } }) => {
+    test.slow();
+    email = login.createEmail();
+    await login.clearCache();
+  });
+
+  test('fails if no user logged in', async ({
+    page,
+    target,
+    pages: { relier },
+  }) => {
+    const query = new URLSearchParams({
+      login_hint: email,
+      return_on_error: 'false',
+    });
+    await page.goto(target.relierUrl + `/?${query.toString()}`);
+    await relier.signInPromptNone();
+
+    //Verify error message
+    expect(await relier.promptNoneError()).toContain('User is not signed in');
+  });
+
+  test('fails RP that is not allowed', async ({ page, pages: { relier } }, {
+    project,
+  }) => {
+    test.skip(
+      project.name !== 'local',
+      'we dont have an untrusted oauth for stage and prod'
+    );
+    const query = new URLSearchParams({
+      login_hint: email,
+      return_on_error: 'false',
+    });
+    await page.goto(`http://localhost:10139` + `/?${query.toString()}`);
+    await relier.signInPromptNone();
+
+    //Verify error message
+    expect(await relier.promptNoneError()).toContain(
+      'prompt=none is not enabled for this client'
+    );
+  });
+
+  test('fails if requesting keys', async ({
+    page,
+    target,
+    pages: { relier },
+  }) => {
+    const query = new URLSearchParams({
+      client_id: '7f368c6886429f19', // eslint-disable-line camelcase
+      forceUA:
+        'Mozilla/5.0 (Linux; Android 6.0; Nexus 5 Build/MRA58N) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.181 Mobile Safari/537.36',
+      // eslint-disable-next-line camelcase
+      keys_jwk:
+        'eyJrdHkiOiJFQyIsImtpZCI6Im9DNGFudFBBSFZRX1pmQ09RRUYycTRaQlZYblVNZ2xISGpVRzdtSjZHOEEiLCJjcnYiOi' +
+        'JQLTI1NiIsIngiOiJDeUpUSjVwbUNZb2lQQnVWOTk1UjNvNTFLZVBMaEg1Y3JaQlkwbXNxTDk0IiwieSI6IkJCWDhfcFVZeHpTaldsdX' +
+        'U5MFdPTVZwamIzTlpVRDAyN0xwcC04RW9vckEifQ',
+      login_hint: email,
```

Comment:
Does this line need a `// eslint-disable-line camelcase` too (re: `login_hint`)?